cldf/requirements.txt

```diff
@@ -0,0 +1,51 @@
+appdirs==1.4.3
+bs4==0.0.1
+certifi==2019.11.28
+chardet==3.0.4
+cldfbench==1.2.2
+cldfcatalog==1.3.0
+clldutils==3.5.0
+colorlog==4.1.0
+csvw==1.7.0
+decorator==4.4.2
+gitdb==4.0.2
+idna==2.9
+isodate==0.6.0
+jdcal==1.4.1
+lingpy==2.6.6
+lingrex==0.1.3
+Markdown==3.2.1
+networkx==2.2
+newick==1.0.0
+numpy==1.18.2
+openpyxl==3.0.3
+packaging==20.3
+pluggy==0.13.1
+purl==1.5
+py==1.8.1
+pybtex==0.22.2
+pycldf==1.14.1
+pyclts==2.0.0
+pyconcepticon==2.6.0
+pycountry==19.8.18
+pyglottolog==3.1.0
+-e git+https://github.com/lexibank/pylexibank.git@1e4a2f54afe7aee2e613f9c4bcb36f7d79f422fc#egg=pylexibank
```

Comment:
I'd go for the latest official release (2.6.0).

Reply:
pylexibank?

Reply:
Yep!